client/my-sites/site-settings/disconnect-site/troubleshoot.jsx

```diff
@@ -14,35 +14,48 @@ import HelpButton from 'jetpack-connect/help-button';
 import JetpackConnectHappychatButton from 'jetpack-connect/happychat-button';
 import LoggedOutFormLinkItem from 'components/logged-out-form/link-item';
 import LoggedOutFormLinks from 'components/logged-out-form/links';
+import { PLAN_JETPACK_FREE } from 'lib/plans/constants';
 import addQueryArgs from 'lib/route/add-query-args';
 import { recordTracksEvent, withAnalytics } from 'state/analytics/actions';
 import { getSiteUrl } from 'state/selectors';
+import { getSitePlanSlug } from 'state/sites/selectors';
 import { getSelectedSiteId } from 'state/ui/selectors';
 
-const Troubleshoot = ( { siteUrl, trackDebugClick, trackSupportClick, translate } ) => (
+const Troubleshoot = ( { isFreePlan, siteUrl, trackDebugClick, trackSupportClick, translate } ) => (
 	<LoggedOutFormLinks>
 		<LoggedOutFormLinkItem
 			href={ addQueryArgs( { url: siteUrl }, 'https://jetpack.com/support/debug/' ) }
 			onClick={ trackDebugClick }
 		>
 			<Gridicon size={ 18 } icon="offline" /> { translate( 'Diagnose a connection problem' ) }
 		</LoggedOutFormLinkItem>
-		<JetpackConnectHappychatButton
-			label={ translate( 'Get help from our Happiness Engineers' ) }
-			eventName="calypso_jetpack_disconnect_chat_initiated"
-		>
+		{ isFreePlan ? (
 			<HelpButton
 				label={ translate( 'Get help from our Happiness Engineers' ) }
 				onClick={ trackSupportClick }
 			/>
-		</JetpackConnectHappychatButton>
+		) : (
+			<JetpackConnectHappychatButton
+				label={ translate( 'Get help from our Happiness Engineers' ) }
+				eventName="calypso_jetpack_disconnect_chat_initiated"
+			>
+				<HelpButton
```

Comment:
I'm noticing that the same `<HelpButton />` code is now repeated twice - once for the free plan case, and once for when HC is unavailable. Would it be better to declare it at the top once and use it for both cases?

Reply:
I was actually considering creating a unified component containing both `<JetpackConnectHappychatButton />` and `<HelpButton />` ([ref](https://github.com/Automattic/wp-calypso/projects/50#card-5335042)) since that seemed quite natural to me (most often, we want to display either, depending on whether an HE is available or not). In edge cases like this, I'd need to continue to conditionally show `HelpButton` (unless we decide the free plan criterion should be streamlined into the generic component). Inclined to leave as-is.